### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vieiramanda11**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi team, great work, but I would like to point out something here you guys are supposed to use a CSS preprocessor for developing the framework, either of SASS or LESS is fine 

**Some important things to do**
Create a class that represents a row
- [x] Classes to define columns sized from 1/12th of the row to 100% of the row
 - [x] The defined columns should have breakpoints
            These breakpoints should be available for different screen sizes.
With breakpoints,  set the webpage you built would be responsive, currently, it is not

**Comment by bolah2009** [comment]
Currently been reviewed

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    width: 160px;
+}
+
+.language-container figure {
+    align-items: center;
+    background-color: white;
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 16px 0px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```
Comment (bolah2009) [inline styles/styles.css]: `SUGGESTION`

Consider using the framework instead, since this CSS property has been defined there.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team :clap: 

The main objective of the project is to ...

> Write the CSS necessary to create a basic 12-column grid framework.

In which you have done a nice job :+1: 

However, to demonstrate it's a **12-column grid framework** I would suggest making the [cloned](https://raw.githack.com/allun26/HTML-CSS-Building-with-css-framework/dev/index.html) page more responsive:

#### Consider using the framework more effectively to achieve this responsiveness. See below:

| Current | Preferred |
| :-----: | :-----: |
| css frame responsive current | css frame responsive preffred |

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Comment by bolah2009** [comment]
I would also suggest using `.gitignore` in the project for files that are not related to the project. Currently, there are **1654** files in this project :open_mouth:. Alternatively, consider using `font-awesome` CDN instead.

**Comment by samgaco** [comment]
### Hi @vieiramanda11 and @allun26 :hand: 

### Your project is approved, you can merge and submit the completion form :+1:. But before doing so, please, address the following:

* You still have to solve the linters checks that have failed.
* Try using a feature-branch for your next projects, read more about it [here]
(https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/)
* **[OPTIONAL]** The point that @bolah2009 made is still not covered:

> #### Consider using the framework more effectively to achieve this responsiveness. See below:
> Current	Preferred
> css frame responsive current	css frame responsive preffred


### Anyways, congratulations on the good work and having your last project completed :clap:

**Pull request merged**